### DISCUSSION TITLE
Default Vehicle Sort for Accounts with Multiple Vehicles

### DISCUSSION THREAD

**Discussion by PlaysWithTrons** [teslamate-org/teslamate#2535] [Ideas]
I recently became a secondary user account for another family member's Tesla. In short, this means my TeslaMate is now tracking two vehicles. The problem is the other family member's car shows up first in the dashboards. I wish there was an easy way to set a default sort order for vehicles so my vehicle could appear at the top of my dashboards.

**Comment by junderw** [comment]
In a similar vein: I rented a Model Y for a month and the owner added me as a driver for that month.

Since default of most of the lists is "Last 30 days" and I haven't driven the Y in over half a year, every time I open my Drives dashboard I have a huge empty panel with the name of the Model Y I drove.

I need to minimize it and then whenever I click on a drive it asks if I want to "save my preferences"... but when I try to save it says "can not save provisioned dashboard"
